CMakeLists.txt

```diff
@@ -16,6 +16,11 @@ include(${CMAKE_SOURCE_DIR}/Installation/cmake/modules/CGAL_SCM.cmake)
 option( CGAL_REPORT_DUPLICATE_FILES "Switch on to start (naive) detection of duplicate source- and headerfiles in packages" OFF)
 
 message( "== CMake setup (DONE) ==\n" )
+
+# Enable testing with BUILD_TESTING
+option(BUILD_TESTING "Build the testing tree." OFF)
+include(CTest)
+
```

Comment:
According to the [CMake Version Compatibility Matrix], the module is shipped since CMake-2.6, and has a compatible interface since then.

[CMake Version Compatibility Matrix]: https://cmake.org/Wiki/CMake_Version_Compatibility_Matrix/StandardCMakeModulesA